modules/@apostrophecms/util/index.js

```diff
@@ -473,13 +476,16 @@ module.exports = {
       // Returns an object like this: `{ object: { ... }, dotPath: 'dot.path.of.object' }`
       //
       // Ignore the `_dotPath` argument to this method; it is used for recursion.
-      findNestedObjectAndDotPathById(object, id, _dotPath) {
+      findNestedObjectAndDotPathById(object, id, { ignoreTemporaryProperties = false } = {}, _dotPath) {
```

Comment:
The code looks good. In A2 docs we call them dynamic properties, which I think is better than "temporary." Let's update this to `ignoreDynamicProperties`. Either way we need to use the same name for the thing everywhere.